packages/trip-details/i18n/en-US.yml

```diff
@@ -20,28 +20,11 @@ otpUi:
       <strong>{bikeMinutes, plural, one {# minute} other {# minutes}}</strong> spent biking during this trip.
       Adapted from <dietaryLink>Dietary Guidelines for Americans 2005, page 16, Table 4</dietaryLink>.
     departure: Depart <strong>{departureDate, date, long}</strong> at <strong>{departureDate, time, short}</strong>
-    flexDropOffMessage: >
-      {hasLeadTime, select,
-        true {To get off at your destination, you must call
-          {hasPhone, select,
-            true {{phoneNumber}}
-            other {ahead}
-          } at least {leadDays, plural, one {# day} other {# days}} in advance}
-        other {You must tell the operator where you want to get off.}
-      }.
-    flexPickupMessage: >
-      To take this route, you must call {hasPhone, select,
-        true {{phoneNumber}}
-        other {ahead}
-      }{hasLeadTime, select,
-        true { at least {leadDays, plural, one {# day} other {# days}} in advance}
-        other {}
-      }.
     title: Trip Details
     tncFare: "{companies} Fare: <strong>{minTNCFare} - {maxTNCFare}</strong>"
     transitFare: Transit Fare
     transitFareEntry: "{name}: <strong>{value}</strong>"
-    tripIncludesFlex: This trip includes flexible routes.
+    tripIncludesFlex: This trip includes flexible routes. {extraMessage}
```

Comment:
Can we just render the extra message separately? (as far as I know `extraMessage` is not subject to i18n.)

Reply:
The reason I did this is to support RTL languages or languages (such as german) where it would be customary to wrap this sort of message in quotes

Reply:
Could you add a comment in `en-US.yml` regarding the quotes?